Deepak_Yadav/P1 Profile/html/about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+
+    <link rel="stylesheet" type="text/css" href="../css/about.css">
+    <link rel="stylesheet" type="text/css" href="../css/style.css">
+    <link rel="stylesheet" href="css/slick.css">
+    <link rel="stylesheet" href="css/slick-theme.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="jquery-3.4.1.min.js"></script>
+    <title>Deepak Yadav</title>
+</head>
+<header>
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <img src="../images/img3.jpg" class="rounded-circle border border-warning" alt="Logo" style="width:60px;">
+        <a class="navbar-brand nav1" href="#"><b>D <span class="text-primary">E</span> EPAK </b></a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item active">
+                    <a class="nav-link" href="home.html"><b>Home</b> <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item active">
+                    <a class="nav-link text-primary" href="about.html"><b>About Me</b> <span
+                            class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item active">
+                    <a class="nav-link" href="acheivement.html"><b>Acheivements</b><span
+                            class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item active">
+                    <a class="nav-link" href="specials.html"><b>Photographs</b></a>
+                </li>
+                <li class="nav-item active">
+                    <a href="#"><i class=" nav-link fa fa-fw fa-search ic1"></i></a>
+                </li>
+            </ul>
+
+        </div>
+    </nav>
+</header>
+
+<body>
+
+
+    <section>
+
```

Comment:
get rid of extra line spaces
missing comments in layout

Reply:
Profile page is updated as per your feedback